scripts/issue/563/emissions_graviaura.py

```diff
@@ -0,0 +1,21 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+
+def main(amount=25000000000000000000000, start_ts=1655481600, end_ts=1658505600):
```

Comment:
We'll need to adjust this to `amount=2000000000000000000000`, `end_ts=1656086400`

Reply:
These parameters can be passed to the script when running it, although it won't hurt to have the default being the correct ones.